compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_markup.cpp

```diff
@@ -118,8 +119,11 @@ static void RenderMmaps(InternalScopedString *buffer,
     // module.base_address == dlpi_addr
     // range.beg == dlpi_addr + p_vaddr
     // relative address == p_vaddr == range.beg - module.base_address
-    buffer->AppendF(kFormatMmap, range.beg, range.end - range.beg, moduleId,
-                    accessBuffer.data(), range.beg - module.base_address());
+    buffer->AppendF(
+        kFormatMmap, reinterpret_cast<void *>(range.beg),
+        static_cast<unsigned int>(range.end - range.beg),
```

Comment:
it's unlucky sanitizer convension, but uptr is equivalend to size_t.

`static_cast<unsigned int>(range.end - range.beg),` is uptr so size format `%z` should be used

So can you try to "{{{mmap:%p:0x%zx:load:%zu:%s:0x%zx}}}";